examples/terraform/aws/output.tf

```diff
@@ -45,9 +45,9 @@ output "kubeone_workers" {
   value = {
     # following outputs will be parsed by kubeone and automatically merged into
     # corresponding (by name) worker definition
-    fra1-a = {
+    pool-a = {
```

Comment:
Can this be potentially backwards incompatible? What if the user provisioned infrastructure using old scripts? Also, if we're doing this change, we should do it for all providers.

Reply:
We don't cover *example* terraform configurations with backward compatibility, and even if we do, we should cancel it. Those are only example show-cases, nothing more.